tests/integration.rs

```diff
@@ -310,3 +341,117 @@ fn usage() -> TestResult {
     trycmd::TestCases::new().case("tests/*.trycmd");
     Ok(())
 }
+
+/// This test is a bit more verbose than the other tests because it uses a slightly
+/// different mechanism for setting up and sweeping / cleaning.
+///
+/// Step-by-step:
+///
+/// * Create a new temporary directory. Other tests only output the built files into
+///   the target directory, but this test copies the whole template project into the
+///   target directory.
+/// * Build the two different (nested) projects using cargo build, but without incremental
+///   compilation, because cargo-sweep doesn't remove files built by incremental compilation,
+///   which affects this test since this test doesn't use a separate target folder for just
+///   the target files.
```

Comment:
This comment is outdated.
```suggestion
/// * Build the two different (nested) projects using cargo build
```

Reply:
Done